packages/mobile/scripts/run_e2e.sh

```diff
@@ -80,16 +83,18 @@ preloadBundle() {
 runTest() {
   reuse_param=""
   if [[ $DEV_MODE == true ]]; then
+    export NUM_RETRIES=0
```

Comment:
leftover?

Reply:
This is behind the `DEV_MODE` flag so that the flakey test runner doesn't run when running tests locally. The problem with it is that when tests are retried, the retry often fails because the state in the device is different than expected, so the error shown is usually not the correct one.

For example, let's say we have screens A and B, and a button in A.
Tests starts, presses button in A and fails while doing some check in B. Now the simulator is in B screen.
When running the retry, the error we see is "button not found in A" because the simulator is in B, but the real error was in the B screen.

Having the flakey test checker disabled in dev mode allows us to get the real error and this is the way I found to do that :)